nsxt/utils_test.go

```diff
@@ -459,6 +458,23 @@ resource "nsxt_policy_tier0_gateway" "test" {
 }`, getAccTestResourceName(), edgeClusterName)
 }
 
+func testAccNsxtPolicyTier1WithEdgeClusterForVPN(edgeClusterName string) string {
+	return fmt.Sprintf(`
+resource "nsxt_policy_tier0_gateway" "test" {
+	display_name = "%s"
+	description  = "Acceptance Test"
+	ha_mode = "ACTIVE_STANDBY"
+}
+resource "nsxt_policy_tier1_gateway" "test" {
+	description               = "Acceptance Test"
+	display_name              = "%s"
+	locale_service {
+		edge_cluster_path = data.nsxt_policy_edge_cluster.%s.path
+	}
+	tier0_path                = nsxt_policy_tier0_gateway.test.path
+}`, getAccTestResourceName(), getAccTestResourceName(), edgeClusterName)
```

Comment:
If I'm not mistaken, using `getAccTestResourceName()` here would mean T0 and T1 names will change between test steps. While this does not affect test result, its an extra update that would increase overall test run time unnecessarily. I suggest assigning random gateway names only once, store these in variables and use those variables here. I believe same issue is present in other VPN tests.